docs/buildah-bud.md

```diff
@@ -749,6 +754,12 @@ buildah bud --no-cache --rm=false -t imageName .
 
 buildah bud --dns-search=example.com --dns=223.5.5.5 --dns-option=use-vc .
 
+### Building an multi-architecture image using a --manifest option (Requires emulation software)
+
+buildah bud --arch arm --manifest myimage /tmp/mysrc
+buildah bud --arch x86_64 --manifest myimage /tmp/mysrc
+buildah bud --arch s390x --manifest myimage /tmp/mysrc
+
```

Comment:
In case it's not in the queue, showing this in action would be a great blog topic.

Reply:
That is the plan.  I think a video of it would be best to really show it working.